docs/semgrep-cloud-platform/getting-started.md

```diff
@@ -94,6 +76,95 @@ Using Semgrep Supply Chain with Semgrep Cloud Platform provides users with the V
 
 </Tabs>
 
+If you are an **administrator** setting up Semgrep for your team, see [Adding an organization](#adding-an-organization).
+
+If you are a **team member joining an existing organization**, see [Joining an organization](#joining-an-organization).
+
+If you are a **personal user**, or simply want to run scans, see [Running scans](#running-scans).
+
+## Adding an organization
+
+This process creates an **organization account**, which your team members in GitHub or GitLab can join. Organization accounts share custom rules that are written by fellow team members, as well as records of findings and scans. 
+
+1. Sign in to Semgrep Cloud Platform. A dialog box appears.
+2. Click **Create an organization**.
+![DESCRIPTION](/img/create-an-org.png#md-width)
+3. Enter your **Organization display name**. This name typically corresponds to the name of your GitHub or GitLab org.
+4. Click **Create**.
+5. Your organization is created and you are prompted to start a scan. Instead of starting a scan, click **<i class="fa-solid fa-gear"></i> Settings** in the navigation menu.
+6. Click **Source code managers**.
+7. Select the source code manager that contains the org to connect to Semgrep Cloud Platform.
+![DESCRIPTION](/img/source-code-manager.png#md-width)
+8. Follow the steps to connect your source code manager. You can review permissions needed by Semgrep in [Requested permissions for GitHub and GitLab](#requested-permissions-for-github-and-gitlab).
+9. Optional: Refer to the instructions in [SSO configuration](/semgrep-cloud-platform/sso/) to set up SSO for your organization.
+
+## Joining an organization
+
+:::info Prerequisites
+- Your admin or person inviting you to Semgrep Cloud Platform should have an existing organization (org) in SCP that is connected to an org in either GitHub or GitLab.

```

Comment:
We may want to avoid use of the word "invite" here because folks are not actually invited. Maybe just "Your admin" should be enough?

Reply:
This is so tricky. I used the term "invite" because that's the language people used to roughly and casually describe their goal, and is the gap we're trying to address because we don't have the functionality in the product per se.

Alternatively, I could remove the above usage of "invite", then explicitly state that Semgrep does not give out email invites, and then instruct admins to tell their team members to sign in after the admin has made an org.

What do you think?

Reply:
Update: Yeah, I went in this direction :)